trace-C19/src/main/webapp/js/script.js

```diff
@@ -1,4 +1,55 @@
 /*
 TODO: Use fetch() to send a POST request at "/outLocation" with a ZIP Code as paramter and 
 you should reive a Json Object with all the locations (lat/long) matching that zip code.
-*/
\ No newline at end of file
+*/
+
+//  const myForm = document.getElementById('zipForm'); 
+
+//  zipForm.addEventListener('submit', function(e) {
+
+//      e.preventDefault();
+
+//      const formData = new FormData(this);
```

Comment:
Nit-pick: I think this may not have worked because of the use of `this`. Instead you could have also said 

```
const formData = new FormData(myFom);
```
`this` can be weird sometimes lol

Reply:
updated and sent again